src/index.js

```diff
@@ -44,11 +44,7 @@ process.env.BUILD = 'production'
 
 export default class Bili extends EventEmitter {
   static generate(options) {
-    try {
-      return new Bili(options).bundle({ write: false })
-    } catch (err) {
-      handleError(err)
-    }
+    return new Bili(options).bundle({ write: false })
   }
 
   static async write(options) {
```

Comment:
maybe remove `try ... catch` for `Bili.write` too, we can handle that in `cli.js` instead.

Reply:
Sounds good, Won't it break anything?

Reply:
OK I go for it or you go?